lighthouse-cli/test/smokehouse/test-definitions/byte-efficiency/byte-config.js

```diff
@@ -5,12 +5,14 @@
  */
 'use strict';
 
+const experimentalConfig = require('../../../../../lighthouse-core/config/experimental-config.js');
+
 /**
  * @type {LH.Config.Json}
  * Config file for running byte efficiency smokehouse audits.
  */
 const config = {
-  extends: 'lighthouse:default',
+  ...experimentalConfig,
```

Comment:
hm, bringing this in is a little weird when we override `audits` anyway, what did we need from the experimental config here? I wouldn't expect us to need this.

Reply:
yeah this is only here while unsized-images is still experimental config to get byte-efficiency to work (modeled after screenshot-config). 

Is there a different way to get audits from default and experimental configs to both run in the same smoketest config?

I was planning to revert this change in a PR immediately after this one that would switch unsized-images to default config & allow the changes in byte-config to be much simpler